po/fr.po

```diff
@@ -5938,19 +5936,19 @@ msgstr "Ignorer les bogues dans votre système"
 
 #: /tmp/fish/implicit/share/completions/apt-listbugs.fish:6
 msgid "Ignore newer bugs than upgrade packages"
-msgstr "Ignorer les nouveaux bugs et mettre à jour les paquets"
+msgstr "Ignorer les nouveaux bogues et mettre à jour les paquets"
```

Comment:
See above for consistency issue with “bogues” versus “bugs”.

Reply:
Consistency issues solved for *bug*.